packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,160 @@
+# Number Field Updates (IxD)
+
+## Overview
+
+### Background
+
+- [Bring number-column formatting options to nimble-number-field](https://github.com/ni/nimble/issues/1855)
+- [Number IxD (Figma)](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1603-3464)
+
+The `nimble-number-field` leverages
+[fast-foundation](https://github.com/ni/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/number-field/number-field.ts)
+for its behavior, so we've inherited much of the behavior, including max/min
+validation and localization.
+
+For supported languages (English, French, German, Japanese, Chinese), the
+_decimal separator_ does not align with user expectations in German. For
```

Comment:
> the _decimal separator_ does not align with user expectations in German.

French, too, right? (I believe Japanese and Chinese use a dot (`.`) like English.)